generator/idl_base.h

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Could you please move these C++ files to other directory? I'd like to put them with `.js` files separately.